script/bootstrap

```diff
@@ -356,20 +348,30 @@ function bootstrapOnUbuntu()
                       freeglut3-dev \
                       libopenal-dev \
                       libsdl-gfx1.2-dev \
+                      libsdl2-gfx-dev \
                       libvorbis-dev \
                       libjpeg-dev \
                       libpng-dev \
                       libgtk-3-dev \
                       libboost-python-dev \
                       libboost-log-dev \
                       libboost-regex-dev \
+                      libboost-program-options-dev \
                       libxmu-dev \
                       clang \
                       lsb-release \
                       python3-pip
       python3 -m pip install --upgrade-strategy eager --upgrade pip
       python3 -m pip install --upgrade-strategy eager cmake
       ;;
+    "impish")
+      echo "Sorry, Ubuntu impish is no longer supported"
+      exit 2
+      ;;
+    "hirsute")
```

Comment:
FYI the `case` functionality can do multi-match via the pipe operator:

```bash
case $X in

"value1"|"value2"|"value3")
    ...
    ;;
"value4"|"value5")
   ...
   ;;
...
esac
```

Reply:
So you would like to combine some of these `case` clauses that essentially do the same thing?